homeassistant/components/light/demo.py

```diff
@@ -53,61 +56,66 @@ def __init__(
         self._effect = effect
 
     @property
-    def should_poll(self):
+    def should_poll(self) -> bool:
         """No polling needed for a demo light."""
         return False
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the light if any."""
         return self._name
 
     @property
-    def brightness(self):
+    def available(self) -> bool:
```

Comment:
Isn't the demo components supposed to be a good examples how to make a proper component? Because if so, I'd keep it here, as a reminder for developers that they should implement available() instead of passing invalid values to the bus (which then mess up recorder etc.)..